docs/source/models/other_models/pregnancy/gbd_2021_mncnh/index.rst

```diff
@@ -0,0 +1,335 @@
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - True
+     -
+   * - Age group start
+     - 10 to 14 years
+     - ID=7
+   * - Age group end
+     - 50 to 54 years
+     - ID=15
+
+
+**Pregnancy modeling steps:**
+
+*At initialization:*
+
+#. Assign pregnancy state according to state prevalence values
+#. Assign partial or full term duration according to table in `Pregnancy term lengths`_ section
+#. Assign sex of infant if pregnancy is full term (stillbirth or live birth)
+#. Assign duration of pregnancy depending on term length and, if applicable, sex of the infant. Note that this is the same value as "gestational age" in other parts of the documentation.
+#. Assign birthweight of simulant child 
+#. Assign propensity values for ANC and ultrasound 
+#. Begin simulation
+
+
+See below for more detailed information on how to assign several of these items. 
+
+.. note::
+
+  In later waves of the model, we will add further items that will need to be assigned at initialization including pre-pregnancy BMI, blood pressure and/or hemoglobin levels. We will add further documentation covering how to assign these items at that time. 
+
+.. todo::
+
+   Add information on assigning ANC and intervention propensities, if correlation is included, etc. 
+
+
+*During simulation:*
+
+Run simulants through the pregnancy model as outlined in the :ref:`MNCNH Portfolio project <2024_concept_model_vivarium_mncnh_portfolio>` page. Record all data outlined on the above page. Then move to the intrapartum model. 
+
+Pregnancy term lengths
+~~~~~~~~~~~~~~~~~~~~~~~
+
+At the beginning of pregnancy, it should be determined whether the pregnancy will be partial term or full term according to the probabilities in the table below.
+
+.. list-table:: Pregnancy term lengths probabilities
+  :header-rows: 1
+
+  * - Term length
+    - Probability
+    - Note
+  * - Partial term
+    - (incidence_c995 + incidence_c374) / (ASFR + ASFR * SBR + incidence_c995 + incidence_c374)
+    - 
+  * - Full term
+    - 1 - probability_partial_term
+    - 
+
+Sex of infant
+~~~~~~~~~~~~~~~
+
+For pregnancies that result in live birth or stillbirth outcomes, infant sex should be determined and recorded acording to the probability of male sex shown in the table below (probability of female birth is equal to 1 minus the probability of male birth). This should be performed at the start of pregnancy (transition from np to p states) or upon initialization into the p state. These sex ratios were calculated using the live births by sex 2020 GBD covariate (ID 1106), `shown here <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/Live%20births%20by%20sex.ipynb>`_. Note that there is no variation by draw in this parameter. 
```

Comment:
Limitation to note, but not to address: only singleton births (no twins, triplets, etc)

Reply:
oh, i see you have noted it below already :)